ai-prompts/aws-cli.txt

```diff
@@ -35,6 +37,7 @@ Best practices
 
 Placeholders
```

Comment:
https://github.com/openops-cloud/openops/pull/571/files#diff-07be703f304e825716966bb3626f9eb4b48aad8b3f2984fe6acf269e28aef569R61

as I wrote on slack, let's use PARAM_NAME instead of <param_name>, in caps lock and without <>

Reply:
ok